### DISCUSSION TITLE
Shading Node Easy Setup

### DISCUSSION THREAD

**Discussion by Kieruss** [StjerneIdioten/I3D-Blender-Addon#126] [Ideas & Feature Requests]
Problem: New people probably don't know how to set up the nodes properly
Real Problem: I have to do stuff, and I'm lazy

Solution: Similar to how EasyNormalMap has an "Add Setup" button that creates a group
![image](https://user-images.githubusercontent.com/24835155/111592634-24eef000-881d-11eb-945b-7d9d1b90c1f0.png)
![image](https://user-images.githubusercontent.com/24835155/111592656-2ae4d100-881d-11eb-8316-8f9d51007539.png)

It would be great if the exporter could do the same thing, plus optionally add default spec/normal maps, and ideally connect all the outputs from the new group to the BSDF

My idea for the group to create
![image](https://user-images.githubusercontent.com/24835155/111592849-6c757c00-881d-11eb-9c2c-a930c4086019.png)

And how it would be linked together (ideally with the right side linked by the addon)
![image](https://user-images.githubusercontent.com/24835155/111592947-8b740e00-881d-11eb-8ecc-49d091c4eb67.png)

**Comment by kbrandwijk** [comment]
I agree. Another source of inspiration could be the SCS Tools, the way they have a shader selection dropdown, and all the nodes are set up to match the in-game display is awesome.
